--- Makefile.in.orig	2006-11-25 18:43:34.000000000 -0800
+++ Makefile.in	2007-05-01 10:17:29.000000000 -0700
@@ -27,26 +27,26 @@
 # unix subdirectory.
 #========================================================================
 
-EXPAT_SOURCES   = xmlrole.c      \
-                  xmltok.c       \
-                  xmlparse.c
-
-TDOM_SOURCES    = xmlsimple.c    \
-                  utf8conv.c     \
-                  dom.c          \
-                  domalloc.c     \
-                  domhtml.c      \
-                  domxpath.c     \
-                  domxslt.c      \
-                  domlock.c      \
-                  tcldom.c       \
-                  nodecmd.c      \
-                  tdominit.c     \
-                  tclexpat.c     \
-                  tdomStubInit.c
+EXPAT_SOURCES   = expat/xmlrole.c      \
+                  expat/xmltok.c       \
+                  expat/xmlparse.c
+
+TDOM_SOURCES    = generic/xmlsimple.c    \
+                  generic/utf8conv.c     \
+                  generic/dom.c          \
+                  generic/domalloc.c     \
+                  generic/domhtml.c      \
+                  generic/domxpath.c     \
+                  generic/domxslt.c      \
+                  generic/domlock.c      \
+                  generic/tcldom.c       \
+                  generic/nodecmd.c      \
+                  generic/tdominit.c     \
+                  generic/tclexpat.c     \
+                  generic/tdomStubInit.c
 
-tdomstub_SOURCES= tdomStubLib.c
-tdom_SOURCES    = $(TDOM_SOURCES) $(EXPAT_SOURCES) @EXTRA_SOURCES@
+tdomstub_SOURCES= generic/tdomStubLib.c
+tdom_SOURCES    = $(TDOM_SOURCES) @EXTRA_SOURCES@
 
 WIN_SOURCES	= 
 UNIX_SOURCES    =
@@ -202,14 +202,14 @@
 
 INCLUDES	= @TDOM_INCLUDES@ @TCL_INCLUDES@ @AOL_INCLUDES@
 
-EXTRA_CFLAGS	= $(MEM_DEBUG_FLAGS) @EXTRA_CFLAGS@ -DBYTEORDER=@BYTEORDER@
+EXTRA_CFLAGS	= $(MEM_DEBUG_FLAGS) @EXTRA_CFLAGS@ @TCL_EXTRA_CFLAGS@ -DBYTEORDER=@BYTEORDER@
 
 DEFS		= $(TCL_DEFS) @DEFS@
 
 CONFIG_CLEAN_FILES = Makefile tdomConfig.sh
 
 CPPFLAGS	= @CPPFLAGS@
-LIBS		= @LIBS@
+LIBS		= -lexpat @LIBS@
 AR		= @AR@
 
 CFLAGS		= @CFLAGS_DEFAULT@ $(EXTRA_CFLAGS) $(SHLIB_CFLAGS) -DTCL_DBGX=$(TCL_DBGX)
