--- configure.orig	2006-11-25 18:43:29.000000000 -0800
+++ configure	2006-11-25 18:50:32.000000000 -0800
@@ -1120,10 +1120,12 @@
 
 	    # check in a few common install locations
 	    if test x"${ac_cv_c_tclconfig}" = x ; then
-		for i in `ls -d ${exec_prefix}/lib 2>/dev/null` \
+		for i in `ls -d ${libdir} 2>/dev/null` \
+		        `ls -d ${exec_prefix}/lib 2>/dev/null` \
 			`ls -d ${prefix}/lib 2>/dev/null` \
 			`ls -d /usr/local/lib 2>/dev/null` \
 			`ls -d /usr/contrib/lib 2>/dev/null` \
+			`ls -d /usr/lib64 2>/dev/null` \
 			`ls -d /usr/lib 2>/dev/null` \
 			; do
 		    if test -f "$i/tclConfig.sh" ; then
@@ -4285,7 +4287,7 @@
 	    # results, and the version is kept in special file).
 
 	    if test -r /etc/.relid -a "X`uname -n`" = "X`uname -s`" ; then
-		system=MP-RAS-`awk '{print }' /etc/.relid'`
+		system=MP-RAS-`awk '{print }' /etc/.relid`
 	    fi
 	    if test "`uname -s`" = "AIX" ; then
 		system=AIX-`uname -v`.`uname -r`
@@ -6715,7 +6717,8 @@
 # Add some private include directories
 #--------------------------------------------------------------------
 
-TDOM_INCLUDES="-I${srcdir}/generic -I${srcdir}/expat"
+#TDOM_INCLUDES="-I${srcdir}/generic -I${srcdir}/expat"
+TDOM_INCLUDES="-I${srcdir}/generic"
 
 #--------------------------------------------------------------------
 # Add optional AOLserver includes
